docs/courses/csintro3/orientation/logic.md

```diff
@@ -1,20 +1,55 @@
-# logic operators
+# Logic Operators
 
-table of blocks -> JavaScript equivalent
+One of the key topics covered with blocks was the idea of using logic to read the state of a program, and alter its behavior accordingly. As expected, everything that we did in blocks, we can do in JavaScript and we can actually do more.
 
-|     |     |
-| --- | --- |
-| a and b | a && b |
-| not a | !a |
+### If and else statements 
 
-etc
+Some of the key blocks we used for logic were the ``||logic:if||`` and ``||logic:if else||`` blocks.
 
-### If and else statements 
+To write these in JavaScript, we simply type
 
 ```typescript
-let num: number = 1;
-if (num == 1) console.log("num is one!");
-else console.log("num is not one!");
+if(condition) {
+	// code for if the condition is true
+}
+```
+
+and
+
+```typescript
+if(condition) {
+	// code for when the condition is true
```

Comment:
I'm indifferent about this. I've seen it both ways. Teaching code comments is literally like a 2-line hint that tells them "Hey, '//' means a comment". Maybe teaching them how to use comments well, would take longer but seems like a csintro4 topic. Plus, if we change it to something like doSomething, it will make it more invalid which will be problematic when we replace "condition" with actual conditionals later in the lesson and our script will still be invalid.

Reply:
Ehh, I think like at least a couple paragraphs is warranted, as otherwise it very well might be skimmed over as the orientation lessons are fairly dense syntax and content wise already. I was gonna do it in the extensions lesson as then we can point to a few in the extension and say "there's a lot of code here that is more than we'd learn, but you can look at the comments to get an idea of what is going on even if you can't read it all", as that gives a reasonable baseline for what comments can look like, which will then be used to guide a lot of making the case study games.

Not sure I see a big distinction in the last part; the code gets changed either way, and it seems a bit easier to say 'replace `doSomething` with `console.log("hello")`' to me at least - a bit more concise / concrete of a tag. If we're saying that you have to change two parts of an if statement to make it (beyond trivially) functional (the condition and the statements inside the curly braces), I think that's actually a decent thing to make sure they think about when they make changes / etc